### DISCUSSION TITLE
FIPS 140-2 - Bring in NIST

### DISCUSSION THREAD

**Discussion by its-a-lisa-at-work** [18F/Security-Compliance#19] [Ideas]
https://github.com/usnistgov/ACVP

Maybe we have someone from NIST also speak to us about all of this as well

**Comment by ohsh6o** [comment]
Probably should just email the people on the Contacts section of [the official ACVP page](https://csrc.nist.gov/Projects/Cryptographic-Algorithm-Validation-Program), Chris Celli. Thoughts?

**Comment by pburkholder** [comment]
I had on my ToDo list for today: "Solve FIPS"

I think what I had in mind was to work with NSA and NIST to make sure open-source distros included FIPS-validated crypto out-of-the-box and by default. Whether that's possible?  I don't know.  But I'm willing to be educated.
